# Executes cartesian pose buffer (synchronized)
---
bool success
string msg
